test/catalog/namespace_handle_test.cpp

```diff
@@ -0,0 +1,50 @@
+#include "catalog/namespace_handle.h"
+#include <algorithm>
+#include <random>
+#include <vector>
+#include "catalog/catalog.h"
+#include "transaction/transaction_manager.h"
+#include "util/test_harness.h"
+namespace terrier {
+
+struct NamespaceHandleTests : public TerrierTest {
+  void SetUp() override {
+    TerrierTest::SetUp();
```

Comment:
You may need to add logger initialization and shutdown calls in your tests.

Reply:
I thought the [test harness](https://github.com/cmu-db/terrier/pull/260/files#diff-a776bc3ea698b0d08a2f6d09c3f25f4cR23) would do this, what should I invoke?